src/rdf4cpp/rdf/storage/node/reference_node_storage/NodeTypeStorage.hpp

```diff
@@ -49,7 +49,7 @@ struct NodeTypeStorage {
 
     struct DefaultBackendTypeHash {
```

Comment:
Is there an alternative? If not, is SelectBackendTypeHash still needed?

Reply:
Yes, if `Backend == SpecializedLiteralBackend<T>` the hasher and equality function are provided by that. Because it needs to be able to unwrap the `std::any` value from a view to be able to compare it to its internal value.